README.md

```diff
@@ -58,11 +58,11 @@ Other features of ArangoDB include:
   application-logic and database together for maximal throughput
 - JavaScript for all: **no language zoo**, you can use one language from your
   browser to your back-end
-- **Flexible data modeling**: model your data as combination of key-value pairs,
+- **Flexible data modelling**: model your data as a combination of key-value pairs,
   documents or graphs - perfect for social relations
 - Different **storage engines**: ArangoDB provides a storage engine for mostly
-  in-memory operations and an alternative storage engine based on RocksDB which 
-  handle datasets that are much bigger than RAM.
+  in-memory operations and an alternative storage engine based on RocksDB which handle
+datasets that are much bigger than RAM.
```

Comment:
80 chars width is not a strict rule for us. Also, you missed to indent the wrapped line properly.